{{cookiecutter.slug}}/frontend.angular/src/styles.scss

```diff
@@ -1,10 +1,4 @@
-@use 'variables' with ( // University Brand Colors
-    $primary: #FFCD00,
-    $green: #24A793,
-    $red: #C00A35,
-    $yellow: #FFCD00,
-    $blue: #5287C6);
-@use 'bulma/bulma';
+@use 'bootstrap';
```

Comment:
Perhaps something like
```scss
// University Brand Colors
$primary: #FFCD00;
$green: #24A793;
$success: $green;
$red: #C00A35;
$danger: $red;
$yellow: #FFCD00;
$warning: $yellow;
$blue: #5287C6;
$info: $blue;
$link-color: $blue;
$link-color-dark: $blue;

@import "bootstrap/scss/bootstrap.scss";
```

To not lose the UU colors (the yellow in the menu is quite something though)